docs/_jekyll/css/customstyles.css

```diff
@@ -79,8 +79,8 @@ h4[id],
 h5[id],
 h6[id],
 dt[id]{
-padding-top: 60px;
-margin-top: -40px
+padding-top: 90px;
+margin-top: -70px
```

Comment:
I'm not sure how we should review changes like this, so I'm just gonna approve it 😄

Reply:
................